cpp/include/cudf/detail/stream_compaction.hpp

```diff
@@ -31,7 +31,7 @@ namespace detail {
  * @copydoc cudf::drop_nulls(table_view const&, std::vector<size_type> const&,
  *                           cudf::size_type, rmm::device_async_resource_ref)
  *
- * @param[in] stream CUDA stream used for device memory operations and kernel launches.
+ * @param[in] stream CUDA stream used for device memory operations and kernel launches
```

Comment:
This line can be removed. Just update the `@copydoc` above here to include the new parameter.
The `@copydoc` will include everything from the public doxygen here. Since the public function did not include a stream the `@param stream` line was added here. Now that the public function includes a stream parameter, this line is no longer needed.
Also, since there is more than one `cudf::drop_nulls` function, the parameters in the `@copydoc` are required so doxygen knows which comments to copy from. If there is only one function like `apply_boolean_mask` below then the parameters in the `@copydoc` tag are not required.
So I think you should be able to remove all the `@param` tags from this file and just update some of the `@copydoc` tags.

Reply:
So this change is only half of what I was requesting. You need to update the `@copydoc` tags here as appropriate. Please read the comment above for more details.

Reply:
Done, thanks @davidwendt